themes/powerline-naked/powerline-naked.theme.sh

```diff
@@ -7,6 +7,7 @@ POWERLINE_LEFT_SEPARATOR=${POWERLINE_LEFT_SEPARATOR:=""}
 
 USER_INFO_SSH_CHAR=${POWERLINE_USER_INFO_SSH_CHAR:=" "}
```

Comment:
> Regarding the powerline-naked theme, I've added a new variable as per the discussion. It's currently not active due to the way the background color parameter is utilized.

Hmm, does it mean `USER_INFO_THEME_PROMPT_COLOR` has been actually used as the text color, and the background color is not specified?

What do you think of describing the situation in the code comment above here? For example, we describe that in the `powerline-naked` theme, the specified background color is used as the text color, that the actual background color is not specified, and that `USER_INFO_THEME_PROMPT_COLOR` is used for the text color. Maybe this description might be wrong since I only quickly checked the code, so please correct it properly in such a case.

Another possibility would be to reverse the background and text colors in the `powerline-naked` theme. Currently, only `${params[1]}` is used for the text color, and `${params[2]}` is ignored by `powerline-naked`, but one could use `${params[2]}` as the background color, and explain to the users in the code comment that the background and text colors are reversed in this theme.

Reply:
Okay, I've been thinking about it.
The whole confusion arises from the fact that I wanted to change the text color in the user's information for the Powerline theme. I accomplished this by using a TEXT_COLOR variable and passing it around as the second argument. This approach was true for most Powerline themes but not for the naked theme. In the naked theme, the first argument already represented the text color. Therefore, a more accurate description of the variable's responsibility, in all cases and not just some, is that it is a secondary color. Consequently, it should be named as such. At least, that's my opinion.

USER_INFO_THEME_PROMPT_SECONDARY_COLOR

What do you think about it?

Reply:
Ah, that is a good idea. +1 for `USER_INFO_THEME_PROMPT_SECONDARY_COLOR`.